www/source/partials/docs/_reference-template-data.html.md.erb

```diff
@@ -6,21 +6,7 @@ These configuration settings are referenced using the [Handlebars.js](https://gi
 
 ## sys
 
-Describes the details of how this specific Supervisor was started
```

Comment:
I think something might me amiss with the template-reference generator. Might exclude this one and I can look into it.